### PR TITLE
Add org ID header

### PR DESCRIPTION
**Pull request by thinkterry**

### CODE REVIEW

File: v1/README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Class | Method | HTTP request | Description
 *RadiusServersApi* | [**RadiusServersList**](docs/RadiusServersApi.md#radiusserverslist) | **Get** /radiusservers | List Radius Servers
 *RadiusServersApi* | [**RadiusServersPost**](docs/RadiusServersApi.md#radiusserverspost) | **Post** /radiusservers | Create a Radius Server
 *RadiusServersApi* | [**RadiusServersPut**](docs/RadiusServersApi.md#radiusserversput) | **Put** /radiusservers:id | Update Radius Servers
+*SearchApi* | [**SearchOrginzationsPost**](docs/SearchApi.md#searchorginzationspost) | **Post** /search/organizations | Search Organizations
```
Comment (lsi1verthorn) [inline v1/README.md]: `searchorganizationspost`? `SearchOrganizationsPost`?

File: v1/api/swagger.yaml [hunk 2]
```diff
@@ -862,6 +969,88 @@ paths:
           enabled: false
           dynamic: false
           statusCode: 200
+  /search/organizations:
+    post:
+      tags:
+      - "Search"
+      summary: "Search Organizations"
+      description: "This endpoint will return Organization data based on your search\
+        \ parameters. This endpoint WILL NOT allow you to add a new Orginization.\n\
```
Comment (lsi1verthorn) [inline v1/api/swagger.yaml]: `Organization`

File: v1/api/swagger.yaml [hunk 3]
```diff
@@ -862,6 +969,88 @@ paths:
...
+        \nYou can use the supported parameters and pass those in the body of request.\
+        \ \n\nThe parameters must be passed as Content-Type application/json.\n\n\n\
+        #### Sample Request\n```\ncurl -X POST https://console.jumpcloud.com/api/search/organizations\
+        \ \\\n  -H 'Accept: application/json' \\\n  -H 'Content-Type: application/json'\
+        \ \\\n  -H 'x-api-key: {API_KEY}' \\\n  -d '{\n  \"search\":{\n    \"fields\"\
+        \ : [\"settings.name\"],\n    \"searchTerm\": \"Second\"\n    },\n  \"fields\"\
+        : [\"_id\", \"displayName\", \"logoUrl\"],\n  \"limit\" : 0,\n  \"skip\" :\
+        \ 0\n}'\n```"
+      operationId: "search_orginzations_post"
```
Comment (lsi1verthorn) [inline v1/api/swagger.yaml]: spelling?

File: v1/api/swagger.yaml [hunk 4]
```diff
@@ -862,6 +969,88 @@ paths:
...
+        200:
+          description: "OK"
+          schema:
+            $ref: "#/definitions/systemslist"
+        401:
+          description: "Unauthorized"
+      security:
+      - x-api-key: []
+      x-stoplight:
+        id: "search_orginzations_post"
```
Comment (lsi1verthorn) [inline v1/api/swagger.yaml]: spelling?

File: v1/api/swagger.yaml [hunk 5]
```diff
@@ -4650,34 +5298,47 @@ x-stoplight:
...
-    multi-tenant-organization-api-header:
-      id: "multi-tenant-organization-api-header"
-      name: "Multi Tenant Organization API Header"
-      content: "The Multi-Tenant Organzation API Header is available for JumpCloud\
-        \ Admins to use when making API requests from Organizations that have multiple\
-        \ managed organizations. \n\nThe `x-org-id` is a required header for all multi-tenant\
+    multi-tenant-organization-api-headers:
+      id: "multi-tenant-organization-api-headers"
+      name: "Multi Tenant Organization API Headers"
+      content: "Multi-Tenant Organzation API Headers are available for JumpCloud Admins\
```
Comment (lsi1verthorn) [inline v1/api/swagger.yaml]: spelling

File: v1/api_client.go [hunk 6]
```diff
@@ -1,7 +1,7 @@
 /*
  * JumpCloud APIs
  *
- * V1 & V2 versions of JumpCloud's API. The previous version of JumpCloud's API. This set of endpoints allows JumpCloud customers to manage commands, systems, & system users.
+ *  JumpCloud's V1 API. This set of endpoints allows JumpCloud customers to manage commands, systems, & system users.
```
Comment (lsi1verthorn) [inline v1/api_client.go]: Is the space supposed to be there at the beginning of the line?

File: v1/docs/SearchApi.md [hunk 7]
```diff
@@ -4,15 +4,59 @@ All URIs are relative to *https://console.jumpcloud.com/api*
 
 Method | HTTP request | Description
 ------------- | ------------- | -------------
+[**SearchOrginzationsPost**](SearchApi.md#SearchOrginzationsPost) | **Post** /search/organizations | Search Organizations
```
Comment (lsi1verthorn) [inline v1/docs/SearchApi.md]: spelling

File: v1/docs/SearchApi.md [hunk 8]
```diff
@@ -4,15 +4,59 @@ All URIs are relative to *https://console.jumpcloud.com/api*
...
 [**SearchSystemsPost**](SearchApi.md#SearchSystemsPost) | **Post** /search/systems | Search Systems
 [**SearchSystemusersPost**](SearchApi.md#SearchSystemusersPost) | **Post** /search/systemusers | Search System Users
 
 
+# **SearchOrginzationsPost**
+> Systemslist SearchOrginzationsPost(ctx, contentType, accept, optional)
```
Comment (lsi1verthorn) [inline v1/docs/SearchApi.md]: spelling

File: v1/search_api.go [hunk 9]
```diff
@@ -26,24 +26,138 @@ var (
...
+ @param contentType 
+ @param accept 
+ @param optional (nil or map[string]interface{}) with one or more of:
+     @param "body" (Search) 
+     @param "fields" (string) Use a space seperated string of field parameters to include the data in the response. If omitted the default list of fields will be returned. 
+     @param "limit" (int32) The number of records to return at once. Limited to 100.
+     @param "xOrgId" (string) 
+     @param "skip" (int32) The offset into the records to return.
+ @return Systemslist*/
+func (a *SearchApiService) SearchOrginzationsPost(ctx context.Context, contentType string, accept string, localVarOptionals map[string]interface{}) (Systemslist,  *http.Response, error) {
```
Comment (lsi1verthorn) [inline v1/search_api.go]: spelling

File: v1/README.md (hunk 1 above)
Comment (jpvajda) [reply v1/README.md]: I'll fix this in another PR, it's just the slug / operator ID which is a stoplight docs thing.

File: v1/api/swagger.yaml (hunk 2 above)
Comment (jpvajda) [reply v1/api/swagger.yaml]: fixed.

File: v1/api/swagger.yaml (hunk 3 above)
Comment (jpvajda) [reply v1/api/swagger.yaml]: corrected.

File: v1/api/swagger.yaml (hunk 4 above)
Comment (jpvajda) [reply v1/api/swagger.yaml]: corrected.

File: v1/api/swagger.yaml (hunk 5 above)
Comment (jpvajda) [reply v1/api/swagger.yaml]: fixed. stoplight doesn't show typos in the text sections.. so annoying!

File: v1/api_client.go (hunk 6 above)
Comment (jpvajda) [reply v1/api_client.go]: that seems just to be how Stoplight renders that section. no space to remove.

File: v1/docs/SearchApi.md (hunk 7 above)
Comment (jpvajda) [reply v1/docs/SearchApi.md]: fixd

File: v1/docs/SearchApi.md (hunk 8 above)
Comment (jpvajda) [reply v1/docs/SearchApi.md]: fixed

File: v1/search_api.go (hunk 9 above)
Comment (jpvajda) [reply v1/search_api.go]: fixed

**Pull request merged**